runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -6502,7 +6515,9 @@ done:;
 								}
 								flattenedFieldClass = VM_VMHelpers::currentClass(flattenedFieldClass);
 							}
-
+							if (J9_ARE_ALL_BITS_SET(flattenedFieldClass->classFlags, J9ClassContainsUnflattenedFlattenables)) {
```

Comment:
Why is this required here? Fetching a field should not be initializing defaults.